The Makefile prune-registry target is out of date and fails.

### Describe the bug

```
docker exec educates-registry registry garbage-collect /etc/docker/registry/config.yml --delete-untagged=true
configuration error: open /etc/docker/registry/config.yml: no such file or directory
Usage:
  registry garbage-collect <config> [flags]

Flags:
  -m, --delete-untagged   delete manifests that are not currently referenced via tag
  -d, --dry-run           do everything except remove the blobs
  -h, --help              help for garbage-collect
  -q, --quiet             silence output
make: *** [prune-registry] Error 1
```

Should perhaps use `educates local registry prune` instead.

### Additional information

_No response_

--- a/Makefile
+++ b/Makefile
@@ -404,7 +404,7 @@ prune-builds:
 	rm -rf project-docs/_build
 
 prune-registry:
-	docker exec educates-registry registry garbage-collect /etc/docker/registry/config.yml --delete-untagged=true
+	docker exec educates-registry registry garbage-collect /etc/distribution/config.yml --delete-untagged=true
 
 prune-all: prune-docker prune-builds prune-registry
 

